org.opent2t.sample.binaryswitch.superpopular/com.smartthings.binaryswitch/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "opent2t-translator-com-smartthings-binaryswitch",
-  "version": "1.0.0",
+  "version": "0.2.0",
```

Comment:
not sure if versions can go down/backward. Avneet mentioned something about this.